src/ex5/server/db/migrations/20220619174307-create-items.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
```

Comment:
because mysql object names are case-insensitive, the convention is snake_case in lower case letters
```suggestion
    await queryInterface.createTable('items', {
```